samples/stock/demo/two-standalone-navigators/demo.js

```diff
@@ -0,0 +1,124 @@
+const commonOptions = {
+    api: {
+        url: 'https://demo-live-data.highcharts.com',
+        access: {
+            url: 'https://demo-live-data.highcharts.com/token/oauth',
+            token: 'token'
+        }
+    }
+};
+
+const AMDISIN = 'US0079031078';
+
+// eslint-disable-next-line no-undef
+const AMDPriceConnector = new Connectors.Morningstar.TimeSeriesConnector({
```

Comment:
I don't know if it's temporary, but I'm currently getting an error here. Error in the utils, but I also posted the files to jsFiddle, and I get the same there: https://jsfiddle.net/highcharts/mL27r05g/

Reply:
Incorrect namespace, changing it makes the JSFiddle work: https://jsfiddle.net/BlackLabel/mbjqwkvn/

```suggestion
const AMDPriceConnector = new HighchartsConnectors.Morningstar.TimeSeriesConnector({
```